tests/unit/data_formats/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (C) 2023 Intel Corporation
```

Comment:
Is this copy-pasted unchanged?

Reply:
Yes, I copy pasted this file without changes